### PR TITLE
Add public network

### PR DESCRIPTION
**Pull request by nvthongswansea**

### CODE REVIEW

File: website/docs/d/publicnetwork.html.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+layout: "gridscale"
+page_title: "gridscale: public network"
+sidebar_current: "docs-gridscale-datasource-public-network"
+description: |-
+  Gets the id of a public network.
+---
+
+# gridscale_public_network
+
+Get date of the public network. This can be used to link public network to a server.
```
Comment (bkircher) [inline website/docs/d/publicnetwork.html.md]: I think you meant 'Get data' here. Anyway, maybe rephrase a bit like

```
Use this to link your servers to the public network easily.
```

File: gridscale/datasource_gridscale_publicnetwork.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		d.Set("network_type", network.Properties.NetworkType)
+		d.Set("location_country", network.Properties.LocationCountry)
+		d.Set("location_iata", network.Properties.LocationIata)
+		d.Set("location_name", network.Properties.LocationName)
+		d.Set("delete_block", network.Properties.DeleteBlock)
+		d.Set("create_time", network.Properties.CreateTime.String())
+		d.Set("change_time", network.Properties.ChangeTime.String())
+		if err = d.Set("labels", network.Properties.Labels); err != nil {
+			return fmt.Errorf("Error setting labels: %v", err)
+		}
```
Comment (bkircher) [inline gridscale/datasource_gridscale_publicnetwork.go:99]: Just a question: is there a reason we're only checking the error when we set the labels here and not for the other properties?

File: website/docs/d/publicnetwork.html.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+## Attributes Reference
+
+The following attributes are exported:
+
+* `name` - The UUID of the network.
+* `location_uuid` - The UUID of the location, that helps to identify which datacenter the network belongs to.
+* `l2security` - Defines information about MAC spoofing protection.
+* `status` - The status of the network.
+* `network_type` - The type of the network.
+* `location_country` - The human-readable name of the country where the network locates.
```
Comment (bkircher) [inline website/docs/d/publicnetwork.html.md]: maybe rephrase: `where the network is located`

**Review by bkircher — CHANGES_REQUESTED** [review]

File: gridscale/datasource_gridscale_publicnetwork.go (hunk 2 above)
Comment (nvthongswansea) [reply gridscale/datasource_gridscale_publicnetwork.go:99]: I'll definitely add error handling block in a new PR. Atm, this error can be skipped (but we should check it in case of worst scenario)

**Review by bkircher — APPROVED** [review]

**Comment by bkircher** [comment]
LGTM
